naga/src/front/wgsl/mod.rs

```diff
@@ -40,9 +40,9 @@ impl Frontend {
     }
 
     fn inner<'a>(&mut self, source: &'a str) -> Result<crate::Module, Error<'a>> {
-        let tu = self.parser.parse(source)?;
+        let mut tu = self.parser.parse(source)?;
         let index = index::Index::generate(&tu)?;
-        let module = Lowerer::new(&index).lower(&tu)?;
+        let module = Lowerer::new(&index).lower(&mut tu)?;
```

Comment:
suggestion: This is the only place where we use `Lowerer::lower`, so if we need to consume something, let's just take it by value. Rust will tell us if we need to do something more complicated.

Reply:
Implemented suggestion as a `fixup!` with  2f57fbd2.